drivers/modem/gsm_ppp.c

```diff
@@ -445,7 +447,7 @@ MODEM_CMD_DEFINE(on_cmd_atcmdinfo_attached)
 
 
 static const struct modem_cmd read_cops_cmd =
-	MODEM_CMD("+COPS", on_cmd_atcmdinfo_cops, 3U, ",");
+	MODEM_CMD_ARGS_MAX("+COPS:", on_cmd_atcmdinfo_cops, 1U, 4U, ",");
```

Comment:
One question, shoudn't it be `1U, 3U`, as previously `3U` was used?

Reply:
Quectel documentation specifies 4 fields to COPS output, normally just 1 is used. But if in future use, like when custom commands list is used, we can get all used.
+COPS: <mode>[,<format>[,<oper>][,<Act>]]